docker/Dockerfile

```diff
@@ -17,6 +17,8 @@ ARG TRAVIS_TAG
 ARG TRAVIS_REPO_SLUG
 ARG TRAVIS_COMMIT
 
+ENV TEST_H5MD ON
```

Comment:
Should we make it an `ARG`?

Reply:
I would simply keep it switched `ON` and test H5MD all the time.

Reply:
I guess then I wouldn’t make a environment variable at all!